components/Releases/ReleaseLayout.jsx

```diff
@@ -56,6 +56,10 @@ export default function ReleaseLayout({
           isSubscribed={isSubscribed}
           isDlcmFriend={isDlcmFriend}
         />
+        <section
+          className={styles.about}
+          dangerouslySetInnerHTML={{ __html: release.about }}
```

Comment:
I would _highly suggest_ you ensure this input is sanitized. Is there another way quill recommends you handle this?

The prop is called `dangerouslySetInnerHTML` for a reason. It's fine to use if you _know_ that the content is clean, but when it's coming from users, you run the risk of XSS and other security issues.

Reply:
added the [dompurify](https://github.com/cure53/DOMPurify) package to handle this.